src/Idris/Parser.idr

```diff
@@ -2065,7 +2065,12 @@ docArgCmd parseCmd command doc = (names, DocArg, doc, parse)
         <|> Keyword <$> anyKeyword
         <|> Symbol <$> (anyReservedSymbol <* eoi
                        <|> parens (Virtual Interactive) anyReservedSymbol <* eoi)
-        <|> Bracket <$> (IdiomBrackets <$ symbol "[|" <* symbol "|]")
+        <|> Bracket <$> (
+              IdiomBrackets <$ symbol "[|" <* symbol "|]"
```

Comment:
IMO it would be nice for `:doc [|` (just the opening bracket) to work and similarly for the other bracket types, if you'd like to make the closing brackets optional while you're in here.

Reply:
I think the lexer/tokeniser disallows unmatched brackets, so that would be much more complicated, and in my opinion not worth it, as the error message it helpful enough to suggest closing the bracket